<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xToolsSocketClientToolUi</class>
 <widget class="QWidget" name="xToolsSocketClientToolUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>165</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="2" column="1">
    <widget class="xToolsSpinBox" name="spinBoxClientPort">
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>44444</number>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="xToolsIpComboBox" name="comboBoxServerAddress">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Client port</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="labelContext">
     <property name="text">
      <string>(closed)</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Server IP</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Server port</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Context</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="xToolsIpComboBox" name="comboBoxClientAddress">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelMessageType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Message</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Client IP</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="xToolsWebSocketMessageTypeComboBox" name="comboBoxMessageType"/>
   </item>
   <item row="8" column="1">
    <widget class="xToolsSpinBox" name="spinBoxServerPort">
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>55555</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="xToolsCheckBox" name="checkBoxSpecifyIpAndPort">
     <property name="text">
      <string>Specify ip and port</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>xToolsCheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">xToolsCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>xToolsSpinBox</class>
   <extends>QSpinBox</extends>
   <header location="global">xToolsSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>xToolsIpComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">xToolsIpComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>xToolsWebSocketMessageTypeComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">xToolsWebSocketMessageTypeComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
